projects/br_scs/src/Mirza/BusinessRegistry/Database/Schema/V0001.hs

```diff
@@ -68,6 +68,7 @@ migration () =
           )
     <*> createTable "users" (UserT
           (field "user_id" pkSerialType)
+          (field "oauth_sub" (varchar (Just defaultFieldMaxLength)) notNull)
```

Comment:
Yep you are right, this is being lazy because migration support doesn't work, so this is what I've been doing up till now, but probably do this properly in the future.